open-reac/src/main/java/com/powsybl/openreac/parameters/input/AbstractElementsInput.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac.parameters.input;
+
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.ampl.executor.AmplInputFile;
+import com.powsybl.commons.util.StringToIntMapper;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+/**
+ * @author Nicolas Pierre <nicolas.pierre at artelys.com>
+ */
+public abstract class AbstractElementsInput implements AmplInputFile {
+
+    private final List<String> elementIds;
+
+    protected AbstractElementsInput(List<String> elementIds) {
+        this.elementIds = elementIds;
+    }
+
+    @Override
+    public InputStream getParameterFileAsStream(StringToIntMapper<AmplSubset> stringToIntMapper) {
+        StringBuilder dataBuilder = new StringBuilder();
+        dataBuilder.append("#amplId powsyblId\n");
+        for (String elementID : elementIds) {
+            int amplId = stringToIntMapper.getInt(getElementAmplSubset(), elementID);
+            String[] tokens = {Integer.toString(amplId), AmplWriterUtils.addQuotes(elementID)};
+            dataBuilder.append(String.join(" ", tokens));
+            dataBuilder.append("\n");
```

Comment:
Do we need to use a portable line separator (System.lineSeparator()) ?
I guess it depends on how AMPL deal with line separators.

Reply:
You are right, we should use the system line seperator. I will check if CRLF works (on windows).

Reply:
Changed it, and still works (I work on windows). Change is pushed